apis/apps/v1alpha1/opsdefinition_types.go

```diff
@@ -46,17 +46,17 @@ type OpsDefinitionSpec struct {
 	// +listType=map
 	// +listMapKey=name
 	// +optional
-	TargetPodTemplates []TargetPodTemplate `json:"targetPodTemplates" patchStrategy:"merge,retainKeys" patchMergeKey:"name"`
+	PodInfoExtractors []PodInfoExtractor `json:"podInfoExtractors,omitempty" patchStrategy:"merge,retainKeys" patchMergeKey:"name"`
```

Comment:
Renaming the field directly may break API backward compatibility?

Reply:
yes, but I will transform them during upgrade kb